tests/integration/istioio/trafficmanagement/trafficshifting/scripts/tcp_trafficshifting_test.txt

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+# $snippet route_traffic_v1.sh
+$ kubectl apply -f @samples/tcp-echo/tcp-echo-all-v1.yaml@ -n istio-io-tcp-traffic-shifting
+# $endsnippet
+
+# $snippet check_tcp_echo_is_running.sh
+$ export INGRESS_HOST={{ .ingressHostCommand }}
+$ export INGRESS_PORT={{ .ingressPortCommand }}
+# $snippet
+
+echo "sending traffic to $INGRESS_HOST:$INGRESS_PORT"
+
+# $snippet send_traffic_to_echo_v1.sh
+$ for i in {1..10}; do \
+docker run -e INGRESS_HOST=$INGRESS_HOST -e INGRESS_PORT=$INGRESS_PORT \
```

Comment:
This could use verification that the prefix `one` is present, indicating that the traffic was routed to the v1 version of the tcp-echo service.

Reply:
Added verification to make sure that no traffic is going to v2 of the service and all traffic is going to v1